airflow/jobs.py

```diff
@@ -212,27 +212,48 @@ def _execute(self):
         raise NotImplementedError("This method needs to be overridden")
 
     @provide_session
-    def reset_state_for_orphaned_tasks(self, dag_run, session=None):
+    def reset_state_for_orphaned_tasks(self, dag_run=None, session=None):
```

Comment:
I also don't like separating the logic everywhere when not necessary for the sake of readability. Too many one line functions is kind of bad